### ISSUE TITLE
[BUG] parameter ignored and pattern only used.

### ISSUE THREAD

**Issue by hoog1511** [schemathesis/schemathesis#1513]
I have a question, in my OAS I have a referenced parameter which has an example and a pattern. When I run the schemathesis test it seems to ignore my example entirely and only generate values using the pattern (not sure if this is a bug or a configuration error on my part). This parameter is inserted in the path of a PUT request and the provided example is the only known existing id. How can I make sure that the example will be used at least once in my generated testcases (preferably the first one)? I know I can force it to be a part of the testcases using enum but that would enforce people to only use the provided values according to documentation (I would like to avoid that). I want to be able to verify both the response of a existing object and a non existing object.

The parameter has the following construction:

```yaml
  /Type/{Type}/Number/{Id}:
    put:
      summary: Update a registration record
      operationId: UpdateRegistration
      deprecated: false
      parameters: 
      
          # path parameters
          - $ref: '#/components/parameters/Id'
     #################################################
components:
  parameters:
    Id:
      name: Id
      in: path
      required: true
      description: The Id of an object
      schema:
        type: string
        example: '000000120816216'
        pattern: '^[0-9]{15}$'
```

**Environment (please complete the following information):**
 - OS: [Windows]
 - Schemathesis version: [latest]
 - Spec version: [Open API 3.0.3]

**Additional context**
I perform the schemathesis tests using the CLI

**Comment by Stranger6667** [comment]
I'd like to change how we work with examples as there are way too many issues with that. I think that it will be reasonable to just generate all combinations for examples. I.e merge them into a single schema-like structure and generate a sample for each distinct parameter, which is linear and hence should be acceptable. All the missing parts could be still filled with random data

**Comment by Stranger6667** [comment]
This is likely was fixed before, but I added a test case in #1952
